unified-runtime/source/adapters/level_zero/image_common.cpp

```diff
@@ -1177,7 +1180,6 @@ ur_result_t urBindlessImagesSampledImageHandleDestroyExp(
     ur_context_handle_t hContext, ur_device_handle_t hDevice,
     ur_exp_image_native_handle_t hImage) {
   // Sampled image is a combination of unsampled image and sampler.
-  // Sampler is released in urSamplerRelease.
```

Comment:
Maybe we should update this comment saying that the sampler is tied to the image on creation, and is destroyed together with the image.

Reply:
Will certainly benefit future reading of the implementation, yep. Will do.